### PR TITLE
'Footer_Form_Microanimation'

### PR DESCRIPTION
**Pull request by shreyanshu07**
Here I have added Microanimation from the below link
https://codepen.io/MichaelArestad/pen/ohLIa
I have matched the color theme with the website and checked the reponsiveness was same .
image_2021-06-28_204535
image_2021-06-28_204629

### CODE REVIEW

**Comment by smaranjitghose** [comment]
## Feedback:
- Please have a look at the entire micro animation.
- The color of the labels i.e. Name, Email, and Message (this missing) should be the same as that of the Send Button.
- Unresponsive for certain mobile screens:

Mobile_Footer

File: assets/css/Check.css [hunk 1]
```diff
@@ -0,0 +1,115 @@
+@import "compass/css3";
+ @import url(https://fonts.googleapis.com/css?family=Open+Sans:400,700,600,300,800);
+
```
Comment (smaranjitghose) [inline assets/css/Check.css:3]: What is this file for?

File: assets/css/Check.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+	 color: #aaa;
+}
+ .skinnyx:focus + label, .skinnyx:active + label {
+	 transform: translateX(-100%);
+}
+ .skinnyx:focus + label:after, .skinnyx:active + label:after {
+	 transform: translateX(100%);
+     
+}
+#yourName {
```
Comment (smaranjitghose) [inline assets/css/Check.css:108]: Name the variables judiciously!

File: assets/js/app.js [hunk 3]
```diff
@@ -135,10 +135,19 @@ let footer = $(`
               </div>
                 <form>
```
Comment (smaranjitghose) [inline assets/js/app.js:136]: The above files are renamed without any requirement leading to error in other sections

File: assets/js/app.js [hunk 4]
```diff
@@ -135,10 +135,19 @@ let footer = $(`
...
                   <form action="https://formcarry.com/s/S2thQbCqEvW" method="POST" accept-charset="UTF-8" >
-                  <input type="text" name="field1" placeholder="Your Name" required/>
-                  <input type="email" name="field2" placeholder="Email Address"  required/>
+                    <link rel="stylesheet"  href="assets/css/Check.css">
+                  <div class="row1">
+  <span>
```
Comment (smaranjitghose) [inline assets/js/app.js:140]: Use a Linter please. This is poorly written

File: assets/js/app.js [hunk 5]
```diff
@@ -135,10 +135,19 @@ let footer = $(`
...
+    <input class="skinnyx" id="yourName" type="text" placeholder="Enter your Name" /><label for="Name">Name</label>
+  </span>
+  <span>
+    <input class="skinnyx" id="yourEmail" type="email" placeholder="Enter your Email-id" /><label for="EmailId">Email</label>
```
Comment (smaranjitghose) [inline assets/js/app.js:144]: Name the classes judiciously!

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by shreyanshu07** [comment]
I have made the following changes :-
1. Changed the names of variables , classes
2. Gave proper indentation in the code
3. Removed the unwanted header files in my check.css file
4. Changed the color of label according to the requirements.
5. Added the same to the "Message" field

**Comment by shreyanshu07** [comment]
@deeqakkk 
image

**Comment by shreyanshu07** [comment]
image
@deeqakkk I think now it's perfect :-)

**Comment by smaranjitghose** [comment]
Still not mobile responsive

**Comment by shreyanshu07** [comment]
image
@smaranjitghose

**Pull request closed without merging**